### ISSUE TITLE
Modern UI components?

### ISSUE THREAD

**Issue by IngwiePhoenix** [fltk/fltk#229]
Hey there!

Been a while since I sat down to look at FLTK. But after looking at a good lot of GUI libraries, I end up back here... Not a bad thing, but rather unexpectedly.

So, the reason I came back is because I am looking into writing a small GUI app for PlayStation 3 save data manipulation - and since it's a little bit tricky, the current programs out there won't do, since a bit of logic is involved (long story short: when one counter is increased, two others have to be decreased accordingly. Current tools don't handle this scenario).

But looking at the default FLTK scheme is, well, odd. I have grown very used to dark themes, toggle buttons, odd-even row colouring and grouped lists of options and the likes. As for the colors, I know I can do with a theme (i think it was called a "scheme" in FLTK?) but toggle switches are a new component entirely.

How would I go about implementing a toggle switch in place of a checkbox - and align it to the right, whilst keeping a nice list view?

Basically: Qt is way too overkill, Electron's runtime cost is gigantic, wxWidgets relies on old Win32 APIs that do not have a way to access the modern components (although [Project Reunion](https://github.com/microsoft/ProjectReunion) might do something there) and currently I don't have a great overview as for the situation on macOS and GTK. I really just want to provide a sleek, flat coloured UI in coherence to what modern UIs these days look like.

Kind regards, Ingwie

**Comment by Albrecht-S** [comment]
Hi and welcome "back", but this is really the wrong place to ask questions about the FLTK library or its usage. Therefore I must close this issue. See below for where to go. Discussions about development of the library itself would also be accepted in fltk.coredev, but as I understand your question is more about usage.

**Standard reply follows:**

This is the wrong place to ask questions about building or running programs with FLTK (user support). FLTK "issues" are reserved for bug reports and requests for enhancements, for further information please visit https://www.fltk.org/bugs.php.

Please ask user questions in our user group [fltk.general](https://groups.google.com/forum/#!forum/fltkgeneral). Your question will have a wider audience and you will likely get more and quicker replies.

Please include the system/platform (Windows, Linux, macOS), the build system (Visual Studio, CMake, Ninja), compiler (gcc, clang), compiler version, and build error messages in text form (copy and paste preferred). TIA.
